packages/calendar/src/LionCalendar.js

```diff
@@ -165,12 +165,12 @@ export class LionCalendar extends LocalizeMixin(LitElement) {
     return this.__focusedDate;
   }
 
-  goToNextMonth() {
-    this.__modifyDate(1, { dateType: 'centralDate', type: 'Month', mode: 'both' });
+  goToNextDate(type) {
```

Comment:
I think having 2 separate functions for this would be a nicer user api? 🤔

e.g. for users it would be
- `el.goToNextMonth()`
- `el.goToNextYear()`

brings nicer autocomplete and you can't get it "wrong" if you would for example pass a type `'asdf'`

Reply:
On top of that, we should not change public method names :)